src/main/java/DeadLine.java

```diff
@@ -3,7 +3,6 @@
 import java.time.format.DateTimeFormatter;
 
 public class DeadLine extends Task {
-    protected String by;
     protected LocalDate bydate;
     protected LocalTime bytime;
```

Comment:
The variable names should be changed to something more understandable and descriptive.

Reply:
Thanks, I missed out on that.